README.md

```diff
@@ -64,81 +91,114 @@ Rule change interval:
+**[Regular transaction tree](https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/dgjsyxd):** 
+The normal transactions in a block: sending DCR to an address, coinbase transactions such as PoW Miner reward and Treasury stipend.
 
-**[Stake transaction tree](https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/dgjsyxd)**: Transactions relating to ticket buying and ticket voting rewards.
+**[Stake transaction tree](https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/dgjsyxd)**: 
+Transactions relating to ticket buying and ticket voting rewards.
 
-**Mempool**: Transactions waiting to be mined.
+**Mempool**: 
+Transactions waiting to be mined.
 
 **Block header:** 
+A 120 byte (when serialized) header used to identify a particular block, which is hashed repeatedly to create proof of work for mining rewards. (SETH: review for accuracy)
 
-**Stakepool:** see Voting Service Provider
+**Stakepool:** 
+see Voting Service Provider
 
-**Voting Service Provider:** Non-custodial services that can be authorized to vote on behalf of a ticket, usually providing a number of geographically distributed servers to reduce the chance of missed tickets.
+**Voting Service Provider:** 
+Non-custodial services that can be authorized to vote on behalf of a ticket, usually providing a number of geographically distributed servers to reduce the chance of missed tickets.
 
-**Staking:** Colloquial term for time-locking DCR in exchange for tickets.
+**Staking:** 
+Colloquial term for time-locking DCR in exchange for tickets.
 
-**Voters:** People who buy tickets and vote with them.
+**Voters:** 
+People who buy tickets and vote with them.
 
-**Treasury:** The [Decred Treasury](https://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx) holds funds for use in development of the project, decisions about how to use these funds are made through Politeia proposals and voting.
+**Treasury:** 
+The [Decred Treasury](https://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx) holds funds for use in development of the project. Decisions about how to use these funds are made through Politeia proposals and voting.
 
-**[Politeia](https://docs.decred.org/governance/politeia/):** System for facilitating the [submission and discussion of proposals](https://proposals.decred.org/) in an environment with transparent censorship. 
+**[Politeia](https://docs.decred.org/governance/politeia/):** 
+A system for facilitating the [submission and discussion of proposals](https://proposals.decred.org/) in an environment with transparent censorship. 
 
-Proof-of-Work mining:
+**Proof-of-Work mining:**
+The process by which Proof-of-Work (PoW) miners validate blocks of transactions. In this process, miners compute hashes until the block header is below the target difficulty. (SETH: review for accuracy. Expand?)
```

Comment:
Agreed. I spent some time searching for other definitions out there, and understanding PoW for Bitcoin (basically the same). Believe this wording came from another definition, and jelled enough with my understanding to reuse it. But this should be reviewed for accuracy by someone that understands this better, and perhaps we can come up with better wording.

Reply:
I generally agree. However, sometimes I will put stuff out for review that I know is wrong. Partly because a wrong guess sometimes is an efficient way of expressing what you're looking for. The "scaffolding" of a correct answer, which does some legwork for the accurate answer. Also, sometimes it's the only way to get a developer's attention. They can ignore a request for review (overloaded as they are), but an inaccurate definition cannot stand. It's like waving a red cape in front of a bull. They lose control. Before you know it you have a wall of text and are hopefully still in one piece. Try and avoid that as much as possible though. I think through some additional research I've been able to get 2/3 of the definitions over the line (confident enough to put in PR for review). PoW is too complex. I say let's leave it out for now.